distro/configs/openmrs/initializer_config/liquibase/liquibase.xml

```diff
@@ -40,4 +40,11 @@
         ]]>
         </sql>
     </changeSet>
+    <changeSet author="siddharth" id="17092025">
+        <insert tableName="privilege">
+            <column name="privilege" value="View OrderTemplates"/>
+            <column name="description" value="Able to view OrderTemplates"/>
+            <column name="uuid" value="53b2f66a-8158-4e10-8bfa-11547cc59f26"/>
+        </insert>
+    </changeSet>
```

Comment:
Can you add this via privilege domain instead of Liquibase

Reply:
I've done this for Ozone HSC, see the [commit](https://github.com/CRUDEM/ozone-hsc/commit/a1e3a30d033baff8a7d7e96687668d9b29ec7262)

Reply:
Done. Please approve